tests/integration/vault_kv_requirer_operator/charmcraft.yaml

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python3
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+type: "charm"
+bases:
+  - build-on:
+    - name: "ubuntu"
+      channel: "22.04"
+    run-on:
+    - name: "ubuntu"
+      channel: "22.04"
+
+parts:
+  charm:
+    build-packages:
+      - cargo
+      - libffi-dev
+      - libssl-dev
+      - pkg-config
+      - rustc
```

Comment:
Since this is only used for tests, can we use binary packages instead, it will help speed up the CI.

Reply:
Done